container/dockershell.sh

```diff
@@ -1,32 +1,55 @@
-#!/bin/bash -x
+#!/bin/bash
 
 # path within container
 SETUP_PATH="/root/db2eventstore-IoT-Analytics/container/setup"
 USER_VOLUME=${HOME}/eventstore_demo_volume
 
+# Deployment type constants, default to cp4d
+declare -r deployTypeCp4d="cp4d"
+declare -r deployTypeDsx="dsx"
+declare -r deployTypeStandalone="standalone"
+declare -r defaultDeployType=$deployTypeCp4d
```

Comment:
I would prefer this to be standalone, no ? but you are thinking temporarily cp4d would be better ?

Reply:
I switched it twice :)  I do think cp4d is the best bet for now, but I agree once we publish the standalone container I'd switch it to that.  Note that the applications still need work i.e. they don't work for standalone as is, so also once that's fixed too.